lib/schema.js

```diff
@@ -337,9 +352,10 @@ function FieldSchema(name, type) {
 
     this.name = name;
     this.type = type;
+    this.default = defaultValue;
 }
 
-//util.inherits(FieldSchema, Schema);
+util.inherits(FieldSchema, Schema);
```

Comment:
Should FieldSchema in fact inherit from Schema?  To me, the things that inherit from Schema are valid types based upon the Avro spec.  There is no such thing as a Field type.  Why don't we just rename FieldSchema to Field and have it not inherit from Schema?